extensions/reviewed/PlaygamaBridge.json

```diff
@@ -3829,17 +3768,17 @@
       "objectGroups": []
     },
     {
-      "description": "Is Leaderboard Get Score Supported.",
-      "fullName": "Is Leaderboard Get Score Supported",
+      "description": "Leaderboards Type = Native.",
+      "fullName": "Leaderboards Type = Native",
```

Comment:
This does not look like it is fitting our naming conventions, user visible text should use natural language and no abbreviations like the equal sign, something like `The leaderboard is of type "Native"` might be more fitting.

Additionally, instead of making a condition for each possible value of the variable, maybe consider a function of the type "Expression and Condition" with a "String from a list of options" return type, this would simplify things a lot, keep things flexible, and be more idiomatic.